implementations.md

```diff
@@ -19,3 +19,21 @@ If you know of any associated projects that are not listed here, please file a p
 ## Testing
 
 * [huawei-openlab/oct](https://github.com/huawei-openlab/oct) - Open Container Testing framework for OCI configuration and runtime
+
+## Hooks
+
+For addressing container issues that are outside the scope of this specification, the following hooks may be useful.
+
+### Linux
+
+#### Networking
+
+* [CNI][], the Container Network Interface.
+  The [`exec-plugins.sh`][cni-exec-plugins.sh] script can be used in both [prestart][] (with `add`) and [poststop][] (with `del`) hooks insert and remove a container from networks [configured][cni-conf] in a host-side directory like `/etc/cni/net.d`.
```

Comment:
On Tue, Jan 26, 2016 at 10:47:43AM -0800, Vish Kannan wrote:
> minor nit: Can we avoid describing the usage of plugins here and
> instead just mention `CNI` as an option?  I'm thinking if this PR
> will become a reason for other net plugin vendors to include their
> specification here.

I think this PR is on hold until someone has time to translate it into
a wiki page [1].  But personally, I'm +1 to net-plugin vendors pushing
docs (ideally, links to external docs) that describe how to setup OCI
hooks for their plugins.  Having a collection of references to the
various net-plugin options would help avoid confusion like [2].

[1]: https://github.com/opencontainers/specs/pull/271#issuecomment-165669973
[2]: https://groups.google.com/a/opencontainers.org/d/msg/dev/9VMBA2dAeuU/_tlRFaNTCgAJ
     Subject: no vNIC in the container, what is a good default .json template?
     Date: Thu, 10 Dec 2015 12:18:07 -0800 (PST)
     Message-Id: <acbb4587-8d80-452a-8908-446f14a5107c@opencontainers.org>